tensorflow/python/keras/backend.py

```diff
@@ -1479,7 +1479,8 @@ def random_uniform_variable(shape, low, high, dtype=None, name=None, seed=None):
 
   Example:
 
-  >>> kvar = tf.keras.backend.random_uniform_variable((2,3), 0, 1)
+  >>> kvar = tf.keras.backend.random_uniform_variable(shape=(2,3),
+  low=0.0, high=1.0)
```

Comment:
I think this is breaking tf_docstring test. You shouldn't wrap the line here.

Reply:
Hi, I've added prepending `... ` so that doctest could recognize it.